### PR TITLE
Update skimage commands

### PR DESCRIPTION
**Pull request by f-amerehi**

### CODE REVIEW

**Comment by f-amerehi** [comment]
Hi @rgeirhos, 
I hope that all is well. I have also fixed issues on Eidolon and have packaged it for installation via `pip install git+https://github.com/f-amerehi/Eidolon.git` (although this isn't included in this PR). Should I include it to make using this repository easier? 
Thanks

File: code/calculate_mean_power_spectrum.py [hunk 1]
```diff
@@ -1,17 +1,15 @@
 #!/usr/bin/env python3
+from typing import Optional
```
Comment (rgeirhos) [inline code/calculate_mean_power_spectrum.py]: Can this be removed?

**Comment by rgeirhos** [comment]
Hi @f-amerehi thanks so much for your pull request, that's very much appreciated!
I just have a single question regarding whether 'from typing import Optional' can be removed; happy to merge otherwise.

**Comment by rgeirhos** [comment]
Regarding the Eidolon repository, perhaps you could add a sentence or two to https://github.com/rgeirhos/generalisation-humans-DNNs#code describing your fix and linking to your repository? I think it's best to keep the Eidolon repo separated, but it might be helpful for future readers to know what you've fixed and we could link your repo from there if you like?

File: code/calculate_mean_power_spectrum.py (hunk 1 above)
Comment (f-amerehi) [reply code/calculate_mean_power_spectrum.py]: Hi again @rgeirhos hope that all is well. Done.

**Comment by f-amerehi** [comment]
Yes, perhaps adding a line ease up applying Eidolon too. For [Eidolon](https://github.com/f-amerehi/Eidolon), I fixed some bugs. And one can install it via: "pip install git+https://github.com/f-amerehi/Eidolon.git".

Thank you very much!

**Comment by rgeirhos** [comment]
Thanks again for the PR, much appreciated!

**Comment by f-amerehi** [comment]
Thank you very much for the great paper and codes. Have a good day!

**Pull request merged**